src/Data/Fin/Properties.agda

```diff
@@ -530,6 +530,35 @@ splitAt-≥ (suc m) (suc i) (s≤s i≥m) = cong (Sum.map suc id) (splitAt-≥ m
 +↔⊎ : ∀ {m n} → Fin (m ℕ.+ n) ↔ (Fin m ⊎ Fin n)
 +↔⊎ {m} {n} = mk↔′ (splitAt m {n}) (join m n) (splitAt-join m n) (join-splitAt m n)
 
+------------------------------------------------------------------------
+-- quotRem
+------------------------------------------------------------------------
+
+-- Fin (m * n) ↔ Fin m × Fin n
+
+quotRem-combine : ∀ {n k} x (y : Fin n) → quotRem k (combine x y) ≡ (x , y)
+quotRem-combine {suc n} {k} y 0F rewrite splitAt-inject+ k (n ℕ.* k) y = refl
+quotRem-combine {suc n} {k} x (suc y) rewrite splitAt-raise k (n ℕ.* k) (combine x y) = cong (Data.Product.map₂ suc) (quotRem-combine x y)
+
+combine-quotRem : ∀ {n} k (i : Fin (n ℕ.* k)) → uncurry combine (quotRem {n} k i) ≡ i
+combine-quotRem {suc n} k i with splitAt k i | P.inspect (splitAt k) i
+... | inj₁ j | P.[ eq ] = begin
+  join k (n ℕ.* k) (inj₁ j)      ≡˘⟨ cong (join k (n ℕ.* k)) eq ⟩
+  join k (n ℕ.* k) (splitAt k i) ≡⟨ join-splitAt k (n ℕ.* k) i ⟩
+  i                              ∎
+  where open ≡-Reasoning
+... | inj₂ j | P.[ eq ] = begin
+  raise {n ℕ.* k} k (uncurry combine (quotRem {n} k j)) ≡⟨ cong (raise k) (combine-quotRem {n} k j) ⟩
+  join k (n ℕ.* k) (inj₂ j)                             ≡˘⟨ cong (join k (n ℕ.* k)) eq ⟩
+  join k (n ℕ.* k) (splitAt k i)                        ≡⟨ join-splitAt k (n ℕ.* k) i ⟩
+  i                                                     ∎
+  where open ≡-Reasoning
+
+------------------------------------------------------------------------
+-- Bundles
+*↔× : ∀ {m n} → Fin (m ℕ.* n) ↔ (Fin n × Fin m)
```

Comment:
I get why this is flipped. But this doesn't match what's advertised at the top... and is also quite counter-intuitive. It would make a lot more sense to keep this name for the version obtained via post-composing this with `swap`.

Reply:
Yes, I would define `remQuot` and prove everything in terms of that. Unsure whether it should be defined in terms of `quotRem` or whether we should redefine it and deprecate `remQuot` entirely? Thoughts?

(`remQuot` is a relatively new addition so deprecating it shouldn't cause too much pain)

Reply:
I've added `remQuot` in terms of `quotRem`. I don't think we should remove `quotRem`, I think it's a natural thing to look for.